File: DAM2/DI/Examen2/src/main/java/org/example/examen2/model/Usuario.java
package org.example.examen2.model;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class Usuario {
    private int id;
    private String nombre,correo,pass;

}

File: DAM2/Android/Tema5/Tienda/app/src/main/java/com/example/tienda/model/Usuario.kt
package com.example.tienda.model

import java.io.Serializable


data class Usuario(
    var nombre: String?= null,
    var apellido: String?= null,
    var edad: Int?= null,
    var correo: String?= null,
    var password: String? = null): Serializable
